x-pack/solutions/security/packages/ecs-data-quality-dashboard/impl/data_quality_panel/data_quality_details/indices_details/pattern/index_check_flyout/index.tsx

```diff
@@ -220,6 +220,7 @@ export const IndexCheckFlyoutComponent: React.FC<Props> = ({
           {selectedTabId === LATEST_CHECK_TAB_ID ? (
             <>
               <LatestResults
+                checkedAt={indexResult?.checkedAt ?? 0}
```

Comment:
Another thing is on line 236 we alsos have HistoricalResults that also render 2 separate versions of incompatibletab (regular non-legacy and slightly modified version)

here `x-pack/solutions/security/packages/ecs-data-quality-dashboard/impl/data_quality_panel/data_quality_details/indices_details/pattern/index_check_flyout/historical_results/historical_results_list/historical_result/index.tsx` they both render chat action